test/OrchardCore.Tests/Localization/LocalizationManagerTests.cs

```diff
@@ -67,11 +72,49 @@ public void GetDictionarySelectsPluralRuleFromProviderWithHigherPriority()
                 It.IsAny<CultureDictionary>())
             );
 
-            var manager = new LocalizationManager(new[] { _pluralRuleProvider.Object, highPriorityRuleProvider.Object }, new[] { _translationProvider.Object }, _memoryCache);
+            var manager = new LocalizationManager([_pluralRuleProvider.Object, highPriorityRuleProvider.Object], [_translationProvider.Object], _memoryCache);
 
             var dictionary = manager.GetDictionary(CultureInfo.GetCultureInfo("cs"));
 
             Assert.Equal(dictionary.PluralRule, csPluralRuleOverride);
         }
+
+        [Theory]
+        [InlineData("en", "Hello en !")]
+        [InlineData("zh-CN", "你好！")]
+        public async Task TestLocalizationRule(string culture, string expected)
+        {
+            var context = new SiteContext();
```

Comment:
This is a functional test, right?

Reply:
Sorry, I don't understand what you mean. I think this is the closest test to the actual operation.